docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+           "entity_value": { "mm": 1, "hh": 50,"nn": "hr"},
+            "language": "hi"
+        },
+        {
             "detection": "message",
-            "original_text": "mummbai",
-            "entity_value": {
-              "to": false,
-              "via": false,
-              "from": true,
-              "value": "Mumbai",
-              "normal": false
-            }
-          }
-        ]
-      }
-      ```
-
-### pnr
-
--   This functionality calls the PNRDetector class to detect pnr.
-
-- Example:
-
-- - Example 1: 
-
-    - ```python
-      message = 'check my pnr status for 2141215305.'
-      entity_name = 'train_pnr'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:*
-
-      ```python
-      from ner_v1.chatbot.entity_detection import get_pnr
-      output = get_pnr(message=message, entity_name=entity_name,structured_value=structured_value,fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/pnr/?message=check%20my%20pnr%20status%20for%202141215305.&entity_name=pnr&structured_value=&fallback_value=&bot_message='
-      ```
-
-    - *CURL Output:*
-
-      ```json
-      {
-        "data": [
-          {
+            "original_text": "१५ मिनट में",
+            "entity_value": {"mm": "15", "hh": 0, "nn": "df"},
+          "language": "hi"
+        },
+        {
+           "detection": "message",
+           "original_text": "शाम में बारह बजकर ३० मिनट",
+           "entity_value": { "mm": 30, "hh": 12, "nn": "pm"},
+            "language": "hi"
+        }]
+    }
+    
+    ```
+
+    
+
+### 2. Date
+
+> entity_type: date
+
+The Date detector module has the capability to detect various form of dates from text in multiple languages. It can detect date from following patterns:
+
+1. *Day month year format*  - 12 feb 2018, 2nd Jan 2019,  12/11/2019, 12-jan-2019
+2. *Day month* - 12 feb, 12/12
+3. Weekday reference - Comming monday, next sunday
+4. Reference day month - 2nd of next month, 2nd sunday of coming month
+5. Current day reference -  tomorrow, yesterday, day after tomorrow
+
+ Currently date detection support has been provided in different languages - `English`,  `Hindi`, `Marathi`,  `Bengali`,  `Gujrati`, `Tamil`. It also supports latin script of given languages.
+
+> To add support for new languages or to add custom patterns in particular language please go through the Date Detector readme [here](https://github.com/hellohaptik/chatbot_ner/blob/develop/ner_v2/detectors/temporal/date/README.md)
+
+*Note - This module has been updated to v2 version of chatbot_ner*
+
+**API Examples:**
+
+- ***Example 1: Detecting day month format date [English] from user message***
+
+  Use the **timezone** parameter to pass your current timezone to date detection
```

Comment:
Use it how? Where can I find a list of valid values for timezone ?

Reply:
where can I find the list ?

Reply:
The values in `Tz database name` column with Status `Canonical` at https://en.wikipedia.org/wiki/List_of_tz_database_time_zones lists valid timezone values